src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -2981,7 +2981,9 @@ namespace cryptonote
   {
     struct request
     {
+      std::string version; // Storage Server version
```

Comment:
Send major, minor, patch as numbers and don't do any string parsing?

Reply:
OK, but that just means I have to do the parsing on the other side.

Reply:
Oh, version is stored as a string in storage server? I'd recommend that be changed to preprocessor tokens stringified when required, but numbers otherwise.

Reply:
Updated.